h/static/scripts/hypothesis.js

```diff
@@ -1,3 +1,8 @@
+if (window.annotator) {
+    console.warn("Not deploying Hypothes.is, in order to avoid double deployment.");
```

Comment:
I think we should remove this log. It's not likely to be seen by or help a user.